hack/make-rules/test-kubeadm-cmd.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
+
+KUBEADM_PATH="${KUBEADM_PATH:=$(realpath "${KUBE_ROOT}")/cluster/kubeadm.sh}"
```

Comment:
realpath isn't available on non-GNU based systems (mac OS X) and so we shouldn't assume it here.  `hack/lib/init.sh` has a helper that emulates it.